pack/fhv/fhvc.json

```diff
@@ -283,6 +283,174 @@
+    "illustrator": "Rob Laskey",
+    "name": "All is Full of Love",
+    "pack_code": "fhvc",
+    "position": 186,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> – When you draw a weakness: Cancel its revelation effect and discard it.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "The Bliss",
+    "back_text": "<b>If you flipped this act while or after playing the Dawn of the Final Day prelude, read the following</b>\n<blockquote><i>you cannot say why but you know why you are here you are here to watch to see to know to perceive to feel to understand to hold in your heart the truth of the cosmos which is that you are but a mote of dust in the brilliant light of infinity and that a single puff of air will send you spiraling into oblivion a single moment and you will cease to be if you even ever were why did you look into the brilliance why did you stare so long at the shining blinding brightness of the stars</i></blockquote>\nYou must either duffer 1 mental trauma or search the collection for a random basic [[Madness]] weakness and add it to your deck. It does not count toward your deck size. You earn 2 bonus experience.",
+    "code": "10687",
+    "clues": null,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "A beautiful sheen hangs over everything, shimmering in the air around you.",
+    "illustrator": "Rob Laskey",
+    "name": "Dawn of the Final Day",
+    "pack_code": "fhvc",
+    "position": 187,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – Talk to the locals and prepare yourself for the Feast.",
+    "type_code": "act"
+  },
+  {
+    "back_name": "The Sleep",
+    "back_text": "<blockquote><i>Glowing spores hang in the air around the effigy, lighting up the mirrors and bathing the festival in a faerie glow. You breathe in and feel your mouth and throat buzz with a light, tingling sensation. The last thing you hear as you pas out is the sound of lilting flutes and tambourines.</i></blockquote>\nEach investigator is defeated and suffers 1 mental or 1 physical trauma.",
+    "code": "10688",
+    "doom": 6,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "Revelers dance to raucous music while others gorge themselves on rank platters of oily food. Visitors and locals alike collapse in fits of laughter while others claw at their eyes. Each of them weat a wild, gleeful smile.",
+    "illustrator": "Rob Laskey",
+    "name": "Around the Table",
+    "pack_code": "fhvc",
+    "position": 188,
+    "quantity": 1,
+    "stage": 2,
+    "text": "<b>Objective</b> – Confront Mother Rachel! If Mother Rachel is defeated: <b>(→R1)</b>.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "Back in the Fold",
```

Comment:
`"back_flavor": "\"Well done,\" Mother Rachel says to you. Lights dance in the sky overhead as the last of the Vale residents are led dutifully to their place at the table. Some look at you with rueful expressions, rightfully betrayed. As you move to assume your place at the table, Mother Rachel steps in your way.\n\"It is not your place to sit with us. You are our guest of honor,\" she smiles. You feel a sharp blow to the back of your head, and then nothing.",`

Reply:
Please insert above back_name.